Server/serverclient.py

```diff
@@ -0,0 +1,44 @@
+import socket
+import time
+
+time_string = ("%H:%M:%S")
+
+
+def time_now():
+    return time.strftime(time_string, time.localtime())
+
+
+class Server:
+    def __init__(self):
+        self.server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        self.server.bind(('', 9009))
+        print(f'[SERVER STARTED {time_now()}]!')
+
+    def start_listen(self):
+        self.server.listen()
+        print(f"[LISTENING {time_now()}] Server is listening on {socket.gethostname()}")
+
+    @staticmethod
+    def receive_msg(conn):
+        return conn.recv(64)
```

Comment:
why are you reading hardcoded value?

Reply:
Its means that method should receive value as a parameter?

Reply:
You should think about it by yourself. Please provide more clever solution instead of reading hardcoded length of message